vendor/aws-vault/LICENSE

```diff
@@ -0,0 +1 @@
+MIT
```

Comment:
Did you use `make update` to generate these files (`VERSION`, `DESCRIPTION` and `LICENSE`)?